app/src/main/java/ch/epfl/favo/view/tabs/favorList/FavorAdapter.java

```diff
@@ -14,28 +14,29 @@
 
 public class FavorAdapter extends ArrayAdapter<Favor> {
 
-    public FavorAdapter(Context context, ArrayList<Favor> favors) {
-        super(context, 0, favors);
-    }
+  public FavorAdapter(Context context, ArrayList<Favor> favors) {
+    super(context, 0, favors);
+  }
 
-    @Override
-    public View getView(int position, View convertView, ViewGroup parent) {
-        // Get the data item for this position
-        Favor favor = getItem(position);
-        assert favor != null;
-        // Check if an existing view is being reused, otherwise inflate the view
-        if (convertView == null) {
-            convertView = LayoutInflater.from(getContext()).inflate(R.layout.favor_list_item, parent, false);
-        }
-        // Lookup view for data population
-        TextView favorTitle = convertView.findViewById(R.id.title);
-        TextView favorDesc = convertView.findViewById(R.id.desc);
-        // Populate the data into the template view using the data object
-        assert favor.getTitle() != null;
-        assert favor.getDescription() != null;
-        favorTitle.setText(favor.getTitle());
-        favorDesc.setText(favor.getDescription());
-        // Return the completed view to render on screen
-        return convertView;
+  @Override
+  public View getView(int position, View convertView, ViewGroup parent) {
+    // Get the data item for this position
+    Favor favor = getItem(position);
+    assert favor != null;
+    // Check if an existing view is being reused, otherwise inflate the view
+    if (convertView == null) {
+      convertView =
+          LayoutInflater.from(getContext()).inflate(R.layout.favor_list_item, parent, false);
     }
+    // Lookup view for data population
+    TextView favorTitle = convertView.findViewById(R.id.title);
+    TextView favorDesc = convertView.findViewById(R.id.desc);
+    // Populate the data into the template view using the data object
+    assert favor.getTitle() != null;
+    assert favor.getDescription() != null;
```

Comment:
Same thing as above here. In general, we want to avoid using assertions in code in the main package and save it for tests. Try/catch blocks would be more appropriate.

Reply:
Yeah I agree, this is still part of an old merge in the master that I didn't touch apart from reformatting

Reply:
Ahh okay I didn't notice that, Git was telling me these lines were added so I thought it was part of the PR.

Reply:
We should ask the author of the file for the reasons of this and to change it properly.